exec/java-exec/src/test/java/org/apache/drill/exec/record/TestVectorContainer.java

```diff
@@ -110,13 +110,16 @@ public void testContainerMerge() {
     RowSet mergedRs = left.merge(right);
     comparison.verifyAndClear(mergedRs);
 
-    // Add a selection vector. Ensure the SV appears in the merged
-    // result. Test as a row set since container's don't actually
-    // carry the selection vector.
+    // Add a selection vector. Merging is forbidden.
```

Comment:
Maybe this can be changed to "//Merging data with a selection vector is forbidden". As is the comment implies that we are adding a selection vector.